controllers/planController.js

```diff
@@ -1,65 +1,61 @@
 const { Plan } = require('../models/plan')
 
-const stripeSecretKey = process.env.STRIPE_SECRET_KEY;
-const stripe = require('stripe')(stripeSecretKey);
+// const stripeSecretKey = process.env.STRIPE_SECRET_KEY;
+// const stripe = require('stripe')(stripeSecretKey);
 
 module.exports = {
-    getAllPlans: async (req, res) => {
-        let jsonProducts = {};
-        try {
-            const products = await stripe.products.list();
-            jsonProducts.data = [];
-            for (let i in products.data) {
-                let jsonProduct = {};
-                if (products.data[i].active) {
-
-                    jsonProduct.id = products.data[i].id;
-                    jsonProduct.name = products.data[i].name;
-                    jsonProducts.data[i] = await getPricesForProduct(jsonProduct);
-                }
-            }
-            res.send(jsonProducts);
-        } catch (e) {
-            console.log("error " + e.message);
-        }
+    getAllPlans: (req, res) => {
+        Plan.find({}).then(result => {
```

Comment:
this could be simplified-
```
Plan
.find({})
.then(res.send)
.catch(logger.error)
```

Reply:
we'll forward it to a repository like you showed us in class